mrizwan83/bronx-warriors
# Add Carousel Feature

# Description
Used Swiper library to integrate a carousel, added pagitation and navigation based on [Swiper documentation](https://swiperjs.com/react).Need to further enhance images and encapsulate possible image component 

## Main changes explained:
- User can navigate through different images, supports swipe gesture on mobile devices
- Arrow indication for users to know to scroll or click through images
- Added 18 images from[ Warriors facebook page](https://www.facebook.com/TheBronxWarriorsFootball/photos)

## How to test:
1. Check out into current branch `Jordy-Add-gallery`
2. Do `npm install`
3. Do `npm run dev`  

## Note:
- Images still need to be optimized and have a set size
- Im considering a possible zoom in feature to show extra details base on the selected picture


## Picture and Video below

![Carousel1](https://github.com/mrizwan83/bronx-warriors/assets/95154804/c19b1f4f-26ff-434e-a152-608ec81fdc42)
ow 


https://github.com/mrizwan83/bronx-warriors/assets/95154804/60a82f94-5451-4a58-9f38-03f6851b3a3a
